src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -51,7 +52,7 @@
     private StackPane statusbarPlaceholder;
 
     /**
-     * Creates a {@code MainWindow} with the given {@code Stage} and {@code Logic}.
+     * Creates a {@code MainWindowTrackermon} with the given {@code Stage} and {@code LogicTrackermon}.
```

Comment:
Should this be changed to 
``* Creates a {@code MainWindow} with the given {@code Stage} and {@code Logic}.`` instead? We should have removed all the Trackermon suffix classes already.

Reply:
Of course i know we should remove all the Trackmon suffix... obviously i just missed this one.